impl/maven-executor/src/main/java/org/apache/maven/cling/executor/internal/ToolboxTool.java

```diff
@@ -100,8 +100,7 @@ public String artifactPath(ExecutorRequest.Builder executorRequest, String gav,
     }
 
     @Override
-    public String metadataPath(ExecutorRequest.Builder executorRequest, String gav, String repositoryId)
-            throws ExecutorException {
+    public String metadataPath(ExecutorRequest.Builder executorRequest, String gav, String repositoryId) {
```

Comment:
The interface actually throws the exception.  Given this is a public method, I'd rather keep it, as it may forbid derived classes to actually throw the exception.

Reply:
Ah, it's an unchecked exception, so the argument is moot.